CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased
+* Fixed an issue where running **validate** with the `-g` flag would skip some validations for old format files.
```

Comment:
is `old format` = unified? If so, the latter is preferred for clarity.

Reply:
It seems like it is.
<details>
<summary>(for further details)</summary>

```python
    def is_old_file_format(file_path, file_type):
        file_yml = get_yaml(file_path)
        # check for unified integration
        if file_type == FileType.INTEGRATION and file_yml.get("script", {}).get(
            "script", "-"
        ) not in ["-", ""]:
            if file_yml.get("script", {}).get("type", "javascript") != "python":
                return False
            return True

        # check for unified script
        if file_type == FileType.SCRIPT and file_yml.get("script", "-") not in [
            "-",
            "",
        ]:
            if file_yml.get("type", "javascript") != "python":
                return False
            return True
        return False
```
</details>

```suggestion
* Fixed an issue where running **validate** with the `-g` flag would skip validations for unified integrations.
```